docs/dashboards/data-modifiers.md

```diff
@@ -98,3 +98,6 @@ You can check the modifiers in action with the following demos:
 
 #### DataGrid with MathModifier
 <iframe style="width: 100%; height: 700px; border: none;" src="https://www.highcharts.com/samples/embed/dashboards/demo/datagrid-mathmodifier" allow="fullscreen"></iframe>
+
+#### Using CSV data and RangeModifier
+<iframe style="width: 100%; height: 733px; border: none;" src="https://www.highcharts.com/samples/embed/highcharts/data-tools/csv-modifiers" allow="fullscreen"></iframe>
```

Comment:
One thing though. Data tools is only internal to test the data layer implementation. So if you like to link it, you should move the demo into some other demo and without data tools bundle.

Reply:
You can move it to the `dashboards/data/` (create a folder if needed).